irr/include/irrMath.h

```diff
@@ -18,47 +18,38 @@ namespace core
 
 //! Rounding error constant often used when comparing f32 values.
 
-const f32 ROUNDING_ERROR_f32 = 0.000001f;
-const f64 ROUNDING_ERROR_f64 = 0.00000001;
+constexpr f32 ROUNDING_ERROR_f32 = 0.000001f;
+constexpr f64 ROUNDING_ERROR_f64 = 0.00000001;
 
 #ifdef PI // make sure we don't collide with a define
 #undef PI
 #endif
 //! Constant for PI.
-const f32 PI = 3.14159265359f;
-
-//! Constant for reciprocal of PI.
-const f32 RECIPROCAL_PI = 1.0f / PI;
-
-//! Constant for half of PI.
-const f32 HALF_PI = PI / 2.0f;
+constexpr f32 PI = 3.14159265359f;
```

Comment:
shouldn't we rely on M_PI and M_PIl in math.h ?

Reply:
sure.